modules/registry-third-party-registries.adoc

```diff
@@ -14,3 +14,30 @@ creation.
 Refreshing the fetched tags is as simple as running `oc import-image
 <stream>`. When new images are detected, the previously described build and
 deployment reactions occur.
+
+[id="authentication_{context}"]
+== Authentication
+{product-title} can communicate with registries to access private image
```

Comment:
Please check if the lines aren’t being hard-wrapped at 80 characters.

Reply:
Addressed